ui/admin/app/routes/scopes/scope/session-recordings.js

```diff
@@ -43,6 +44,12 @@ export default class ScopesScopeSessionRecordingsRoute extends Route {
         recursive: true,
       });
 
+      // Sort sessions by created time descending (newest on top)
+      const sortedSessionRecordings = sortBy(
+        sessionRecordings,
+        'created_time',
+      ).reverse();
```

Comment:
```suggestion
      const sortedSessionRecordings = orderBy(
        sessionRecordings,
        'created_time',
        'desc',
      )
```

We should just use lodash's [orderBy](https://lodash.com/docs/#orderBy) instead of reversing the array.

Reply:
oh yup that's much easier to read! thank u!